spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,109 @@
+require "rails_helper"
+
+RSpec.describe "Movie Show Page" do 
+  it "shows the movie's title, creation year, and genre" do
+
+    paramount = Studio.create!({name: "Paramount Pictures", location: "Hollywood"})
+    jaws = paramount.movies.create({title: "Jaws", creation_year: "1976", genre: "Action"})
+
+    visit "/movies/#{jaws.id}"
+
+    expect(page).to have_content("Title: Jaws")
+    expect(page).to have_content("Year: 1976")
+    expect(page).to have_content("Genre: Action")
+  end
+
+  it "shows each movie's actors sorted from youngest to oldest" do 
```

Comment:
This test case can just be edited to "shows each movie's actors" since it's not actually testing sorting here (that's below)